libats-http/src/main/scala/com/advancedtelematic/libats/http/ServiceHttpClient.scala

```diff
@@ -28,9 +30,14 @@ trait ServiceHttpClientSupport {
 
 abstract class ServiceHttpClient(httpClient: HttpRequest => Future[HttpResponse])
                                 (implicit system: ActorSystem, mat: Materializer) {
+
```

Comment:
I would move it outside of `ServiceHttpClient`

Reply:
As now, or you had somewhere else in mind?